FreeRTOS-Plus/Demo/FreeRTOS-IoT-Libraries-LTS-Beta2/mqtt/mqtt_plain_text/DemoTasks/PlaintextMQTTExample.c

```diff
@@ -451,35 +552,93 @@ static void prvCreateMQTTConnectionWithBroker( MQTTContext_t * pxMQTTContext,
 static void prvMQTTSubscribeToTopic( MQTTContext_t * pxMQTTContext )
 {
     MQTTStatus_t xResult;
-    MQTTSubscribeInfo_t xMQTTSubscription[ 1 ];
 
     /***
      * For readability, error handling in this function is restricted to the use of
      * asserts().
      ***/
 
     /* Some fields not used by this demo so start with everything at 0. */
-    ( void ) memset( ( void * ) &xMQTTSubscription, 0x00, sizeof( xMQTTSubscription ) );
+    ( void ) memset( ( void * ) pxGlobalSubscriptionList, 0x00, sizeof( pxGlobalSubscriptionList ) );
 
     /* Subscribe to the mqttexampleTOPIC topic filter. This example subscribes to
      * only one topic and uses QOS0. */
-    xMQTTSubscription[ 0 ].qos = MQTTQoS0;
-    xMQTTSubscription[ 0 ].pTopicFilter = mqttexampleTOPIC;
-    xMQTTSubscription[ 0 ].topicFilterLength = ( uint16_t ) strlen( mqttexampleTOPIC );
+    pxGlobalSubscriptionList[ 0 ].qos = MQTTQoS0;
+    pxGlobalSubscriptionList[ 0 ].pTopicFilter = mqttexampleTOPIC;
+    pxGlobalSubscriptionList[ 0 ].topicFilterLength = ( uint16_t ) strlen( mqttexampleTOPIC );
 
     /* Get a unique packet id. */
     usSubscribePacketIdentifier = MQTT_GetPacketId( pxMQTTContext );
 
     /* Send SUBSCRIBE packet. */
     xResult = MQTT_Subscribe( pxMQTTContext,
-                              xMQTTSubscription,
-                              sizeof( xMQTTSubscription ) / sizeof( MQTTSubscribeInfo_t ),
+                              pxGlobalSubscriptionList,
+                              sizeof( pxGlobalSubscriptionList ) / sizeof( MQTTSubscribeInfo_t ),
                               usSubscribePacketIdentifier );
 
     configASSERT( xResult == MQTTSuccess );
 }
 /*-----------------------------------------------------------*/
 
+static void prvUpdateSubAckStatus( MQTTPacketInfo_t * pxPacketInfo )
+{
+    MQTTStatus_t xResult = MQTTSuccess;
+    uint8_t * pPayload = NULL;
+    size_t pSize = 0;
```

Comment:
Shouldn't these follow the FreeRTOS naming convention like the other functions?
```suggestion
    uint8_t * pucPayload = NULL;
    size_t ulSize = 0;
```